float32 target_pos
uint16 move_time